lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -135,6 +168,13 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
     categoriesManagementMode = CategoriesManagementMode.ASSIGN;
     categoryControlVisible = false;
     classifiableChanged = this.classifiableChangedSubject.asObservable();
+    generalInfoPanelState: boolean;
+    tagsPanelState: boolean;
```

Comment:
the word "state" implies some object, you have just a boolean, so "state" is confusing. If you control visibility, just use "isGeneralInfoPanelVisible", "isTagPanelVisible", "isCategoriesPanelVisible" and so on